akka-persistence/src/main/scala/akka/persistence/journal/PersistencePluginProxy.scala

```diff
@@ -84,6 +68,17 @@ final class JournalProxy(config: Config) extends Actor with Stash with ActorLogg
       }
       context.become(active(target, targetAtThisNode = true))
     } else {
+      val targetAddressKey = s"target-${pluginType.qualifier}-address"
+      if (config.hasPath(targetAddressKey)) {
```

Comment:
We normally define the property in the `reference.conf` anyway, with a placeholder value, such as empty string `""`.
In that way we can document the property and it is more visible that it exists.